microsetta_interface/templates/reports.jinja2

```diff
@@ -90,9 +90,23 @@
                 <div class="barcode-col barcode-text">
                     {{ sample.sample_barcode }}
                 </div>
-                <div class="barcode-col barcode-col-end" id="btn-view-{{ sample.sample_id }}">
-                    <span class="sample-label">{{ _('Not Received Yet') }}</span>
-                </div>
+                {% if sample.latest_scan_timestamp is none %}
+                    <div class="barcode-col barcode-col-end" id="btn-view-{{ sample.sample_id }}">
+                        <span class="sample-label">{{ _('Not Yet Received ') }} </span>
+                    </div>
+                {% elif sample.last_update is none %}
+                    <div class="barcode-col barcode-col-end" id="btn-view-{{ sample.sample_id }}">
+                        <span class="sample-label">{{ _('Sample Received - Information Needed ') }} </span>
+                    </div>
+                {% elif sample.latest_scan_timestamp > sample.last_update %}
```

Comment:
The differentiation of 'Information Needed' and 'Information Still Needed' isn't important from a UI perspective. Let's wrap this condition into the above (`elif sample.last_update is none or sample.latest_scan_timestamp > sample.last_update`).

Reply:
A recent timestamp alone doesn't imply more information is needed? It feels like this needs to be coupled with the sample status from scanning, though perhaps that's being done implicitly upstream

Reply:
The samples are already separated based on lock status, this loop is only through the samples that are not locked.